src/index.tsx

```diff
@@ -1,17 +1,29 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import './index.css'
+import App from './App'
+import reportWebVitals from './reportWebVitals'
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import Account from 'components/account/Account'
+import Transaction from 'components/transaction/Transaction'
+import About from 'components/About'
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route path="account" element={<Account />} />
+          <Route path="transaction" element={<Transaction />} />
+          <Route path="about" element={<About />} />
+        </Route>
```

Comment:
Good use of nested routes. However It doesn't make sense for the root route to be just an empty App. That component isn't really built to be standalone as you have it now. You could make /account the root path as well? Or _just_ the root path?
```suggestion
        <Route element={<App />}>
          <Route path="/" element={<Account />} />
          <Route path="account" element={<Account />} />
          <Route path="transaction" element={<Transaction />} />
          <Route path="about" element={<About />} />
        </Route>
```